pkg/integrations/gcp_exporter/gcp_exporter.go

```diff
@@ -0,0 +1,214 @@
+package gcp_exporter
+
+import (
+	"context"
+	"crypto/md5"
+	"encoding/hex"
+	"errors"
+	"fmt"
+	"net/http"
+	"strings"
+	"time"
+
+	"github.com/PuerkitoBio/rehttp"
+	"github.com/go-kit/log"
+	"github.com/prometheus-community/stackdriver_exporter/collectors"
+	"github.com/prometheus-community/stackdriver_exporter/utils"
+	"github.com/prometheus/client_golang/prometheus"
+	"golang.org/x/oauth2/google"
+	"google.golang.org/api/monitoring/v3"
+	"google.golang.org/api/option"
+	"gopkg.in/yaml.v3"
```

Comment:
I think we prefer to use `v2`, for some reason `v3` is giving undesired results when marshaling sometimes